.changeset/ninety-snails-rescue.md

```diff
@@ -0,0 +1,5 @@
+---
+'@clerk/localizations': minor
```

Comment:
```suggestion
'@clerk/localizations': patch
```

Since we are just adding new translations vs a new language, we can mark this as a patch.